### ISSUE TITLE
Inproper Model count on Startup for Chaturbate models

### ISSUE THREAD

**Issue by moooby** [jrudess/mfc_cb.node#15]
Hello, I've noticed a new bug after upgrading to the latest version.  When the application is starting it seems to report the MFC model numbers from config for the CB models.  See example below.

```
[20170424_162814] CB  5 model(s) in config
[04/24/2017 - 16:28:14] Connecting to MyFreeCams chat server xchat32...
[04/24/2017 - 16:28:15] Login handshake completed. Logged in as 'Guest01655' with sessionId 327710953
[20170424_162815] MFC 5 model(s) in config
[20170424_162826] MFC [DEBUG] Done, waiting 120 seconds.
```

If I add or remove MFC models, the CB models count is always identical to the MFC count.

**Comment by moooby** [comment]
The problem is in main.js

```
if (config.enableCB) {
  CB.create(CB);
  common.msg(CB, config.mfcmodels.length + ' model(s) in config');
  mainSiteLoop(CB);
```

It should be:

`config.cbmodels.length`

I'm really new to git and github so I don't know how to contribute fixes officially, (or if I even can) but I just thought I'd point this out.

**Comment by jrudess** [comment]
Thanks for submitting the issue.  My test config.yml just happened to have the same number of MFC and CB models so I missed this.

**Comment by jrudess** [comment]
Released in 0.3.3
